packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -136,21 +139,28 @@ const projectOptions = computed(() => [
 				label: project.name,
 				command: async () => {
 					let response: ProjectAsset['id'] | null = null;
-					let assetName = '';
+					let assetName: string | undefined = '';
```

Comment:
When can it be undefined, when we just set it to `''`?

Reply:
thanks!